packages/react-ui-components/package.json

```diff
@@ -6,7 +6,7 @@
   "main": "./src/index",
   "scripts": {
     "build": "exit 0",
-    "build-standalone-esm": "rm -Rf .postcssrc lib-esm && tsc --build tsconfig.esm.json && tsc --build tsconfig.esmtypes.json && rsync --prune-empty-dirs  -r --include='*/' --include='*.css' --exclude='*' src/ lib-esm -v && node scripts/generate-standalone-postcssrc.js",
+    "build-standalone-esm": "rm -Rf .postcssrc lib-esm && tsc --build tsconfig.esm.json && tsc --build tsconfig.esmtypes.json && rsync --prune-empty-dirs  -r --include='*/' --include='*.css' --include='*.svg' --exclude='*' src/ lib-esm -v && node scripts/generate-standalone-postcssrc.js",
```

Comment:
This should be in a the other PR, right? I guess we can just merge that one and rebase.

Reply:
Yes see my comment. This commit will just disappear then.